experiment/test_config.yaml

```diff
@@ -690,7 +690,7 @@ nodeCommon:
     envs:
     - CLEANUP=true
     - GCE_ZONE=us-central1-f
-    - KUBELET_ARGS=--cgroups-per-qos=true --cgroup-root=/
+    - KUBELET_ARGS="--cgroups-per-qos=true --cgroup-root=/"
```

Comment:
env file should not contain quotes (you can look at other env files, and we have unit-test guard all e2e env files does not have quotes), I think instead we need to fix the place references it